dockerfiles/scylla/environment.yml

```diff
@@ -7,14 +7,27 @@ channels:
 dependencies:
   - python<=3.10
   - pysam
+  - nextflow
   - pip
+  - kraken2
   - kraken2-server # not on mac
   - bracken # needs to be manually installed from git and softlinked if Mac M1
   - fastcat
   - biopython
+  - flye
+  - megahit
   - pandas
-  - numpy
+  - numpy~=1.23.1
   - taxonkit
   - fastp
   - tabix
-  - mako
\ No newline at end of file
+  - mako
+  - spades # test installation
+  - rnabloom
+  - bbmap
+  - prodigal # for virbot
+  - hmmer # for virbot
+  - diamond # for virbot
+  - genomad
+  - pip:
+    - git+https://github.com/rmcolq/RNA_virus_detector.git@dev
```

Comment:
I've raised a separate issue for this. The current setup works within the existing model where everything has been in a single docker container. But agree we might want to split them up to avoid conflicts etc